actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -389,6 +395,7 @@ def []=(name, options)
           options = { value: value }
         end
 
+        preprocess_options(options)
         handle_options(options)
```

Comment:
That was what I meant then… but now I'm more interested in https://github.com/rails/rails/pull/30121#discussion_r133540949 and https://github.com/rails/rails/pull/30121#discussion_r133542338 😊